packages/playground/src/weblets/freeflow.vue

```diff
@@ -76,6 +77,7 @@
           rootFilesystemSize,
           ssdDisks: disks.map(d => d.size),
         }"
+        :has-custom-domain="selectionDetails?.domain?.enabledCustomDomain"
```

Comment:
Why do we pass this to selectionDetails, I couldn't find it on the component props can you explain please?